Test/FMI3/parser_test_xmls/arrays/valid/binary_array/modelDescription.xml

```diff
@@ -7,7 +7,7 @@
 
   <CoSimulation modelIdentifier="VariableTypes"/>
   <ModelVariables>
-    <Binary name="binary_array" valueReference="122" maxSize="28" initial="exact" description="Testing binary arrays">
+    <Binary name="binary_array" valueReference="125" maxSize="28" initial="exact" description="Testing binary arrays">
```

Comment:
125 was an arbitrary change I did to simplify debugging (since there are other tests with vr 122)